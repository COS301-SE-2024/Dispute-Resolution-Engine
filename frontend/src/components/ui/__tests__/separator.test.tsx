import renderer from 'react-test-renderer';
import { Separator } from '../separator';

it('renders correctly', () => {
  const tree = renderer
    .create(<Separator />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
