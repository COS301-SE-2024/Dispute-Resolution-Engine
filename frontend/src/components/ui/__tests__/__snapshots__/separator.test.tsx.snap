// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="shrink-0 bg-zinc-200 dark:bg-zinc-800 h-[1px] w-full"
  data-orientation="horizontal"
  role="none"
/>
`;
